tests/functional/simple_test.py

```diff
@@ -453,57 +453,62 @@ def test_downgrade(tmpdir):
     flake8_older()
 
 
+def utime(path, time):
+    """set both mtime and atime of a py.path object"""
+    from os import utime
+    utime(path.strpath, (time, time))
+
+
 def test_remove_stale_cache_values(tmpdir):
     """Tests that we remove stale (older than a week) cached packages
     and wheels, while still keeping everything created within the past week.
     """
-    import os
     import time
 
     tmpdir.chdir()
-    home_path = str(Path('.').realpath())
+    home_path = Path()
 
-    pip_path = home_path + '/.pip'
-    cache_path = pip_path + '/cache'
-    wheelhouse_path = pip_path + '/wheelhouse'
+    pip_path = home_path / '.pip'
+    cache_path = pip_path / 'cache'
+    wheelhouse_path = pip_path / 'wheelhouse'
 
-    stale_cached_package = cache_path + '/stale_package'
-    fresh_cached_package = cache_path + '/new_package'
+    stale_cached_package = cache_path / 'stale_package'
+    fresh_cached_package = cache_path / 'new_package'
 
-    stale_cached_wheel = wheelhouse_path + '/stale_wheel'
-    fresh_cached_wheel = wheelhouse_path + '/new_wheel'
+    stale_cached_wheel = wheelhouse_path / '/stale_wheel'
```

Comment:
This (and the line below it) shouldn't have the leading / in the string. Though I'm curious if it gets escaped or what exactly happens there.